### ISSUE TITLE
"Open in advanced web interface" text box too large when using default Glitch-Soc flavor, advanced web interface, and narrow screen

### ISSUE THREAD

**Issue by jenbanim** [TheEssem/mastodon#6]
### Steps to reproduce the problem

1. Be logged in
2. Switch to the default glitch-soc flavor
3. Check the "Enable advanced web interface"
4. Navigate to the public/local page (eg. https://wetdry.world/public/local)
5. Narrow the screen until it displays in single-column mode (works on desktop & mobile)
6. A refresh may be necessary

### Expected behaviour

The "Open in advanced web interface" text box should disappear

### Actual behaviour

The "Open in advanced web interface" text box stays visible, and wraps, pushing the sidebar navigation buttons down

### Detailed description

Here is how the UI appears on wetdry.world:

![wet_dry_world_bug](https://github.com/TheEssem/mastodon/assets/40228062/fc3f531b-08e3-4d39-933c-b5c1a900d886)

Here is how the UI appears on infosec.exchange which is running the upstream glitch-soc branch showing the expected behavior

![Infosec Exchange](https://github.com/TheEssem/mastodon/assets/40228062/62c5446c-bc8a-46ec-98f6-6162d4e51b56)

### Mastodon instance

wetdry.world

### Mastodon version

v4.3.0-alpha.0+chuckya

### Browser name and version

All browsers (checked Chrome, Firefox, iOS Safari)

### Operating system

BingusOS

### Technical details

I was able to fix this issue by applying custom CSS to my instance

```CSS
@media screen and (max-width: 1174px) {
  .switch-to-advanced {
    display: none;
  }
}
```

I totally understand if this is out-of-scope since you're using the mastomodern theme by default, just thought I'd throw this out there in case anyone else is running into the same issue. Also if you'd rather just not worry about issues here that's cool just lemme know

**Comment by TheEssem** [comment]
Seems to be related to 5d9df170cf7de0492f0f68519e984bc7bee7e852?

**Comment by TheEssem** [comment]
Fixed with bc5d74c

**Comment by jenbanim** [comment]
Brilliant, thanks!
